app/views/shared/_menu.html.erb

```diff
@@ -10,9 +10,9 @@
         </div>
         <li role="separator" class="mdc-deprecated-list-divider"></li>
       <% end %>
-      <%= drawer_menu_navigation_item('Inicio', root_path) %>
-      <%= drawer_menu_navigation_item('Materias', all_subjects_path) %>
-      <%= drawer_menu_navigation_item('Avance de Carrera', profile_path) %>
+      <%= drawer_menu_navigation_item('Avance de Carrera', root_path) %>
```

Comment:
What about renaming this section to `Trayectoria Sugerida` to keep it consistent with the "[oficial document](https://eva.fing.edu.uy/pluginfile.php/286695/mod_resource/content/3/TrayectoriaSugerida_Ene2021.pdf)"?

Perhaps that's a discussion that has already happened... if that's so, I'm ok with whatever you decide 🙂